src/plugins/bubble/bubble.js

```diff
@@ -0,0 +1,172 @@
+ */
+var componentName = "wb-bubble",
+	selector = "." + componentName,
+	initEvent = "wb-init" + selector,
+	isNotif,
+	$document = wb.doc,
+	i18nDict = {
+		en: {
+			"bubble-skip": "Skip to",
+			"bubble-closeNotif": "Close notification"
+		},
+		fr: {
+			"bubble-skip": "Passer à",
+			"bubble-closeNotif": "Fermer la notification"
+		}
+	},
+
+	/**
+	 * Adjust bubble position to the footer of the current page
+	 * @method adjustBubbleMargin
+	 * @param {jQuery DOM element} $selector Element which is the actual bubble
+	 */
+	adjustBubbleMargin = function( $selector ) {
+
+		var $footer = $( "#wb-info" );
+
+		// Ensure that the bubble does not go passed the footer
+		if ( $footer.length ) {
+
+			// Add some white space over the footer for the bubble to sit
+			$footer.addClass( componentName + "-mrgn" );
+
+			// Keep the bubble sticky while scrolling Y until user reaches the footer
+			var stickyUntilFooter = function( $element ) {
+
+			// Equals to bubble default bottom value in CSS
+				var bottomY = $( "main p" ).eq( 0 ).height() || 30;
+				var $window = $( window );
+
+				if ( $window .scrollTop() >= $document.outerHeight() - $window .outerHeight() - $footer.outerHeight() ) {
+					$element.css( {
+						bottom: ( $footer.outerHeight() - ( $document.outerHeight() - $window.outerHeight() - $window.scrollTop() ) + bottomY )
+					} );
+				} else {
+					$element.css( {
+						bottom: bottomY
+					} );
+				}
+			};
+
+			// Correct bubble positionning on load, on resize an on Y scroll if necessary
+			stickyUntilFooter( $selector );
+
+			$( window ).on( "resize scroll", function() {
+				stickyUntilFooter( $selector );
+			} );
+		} else {
+			$( "main" ).after( "<footer id=\"wb-info\" class=\"" + componentName + "-mrgn" + "\"></footer>" );
+		}
+	},
+
+
+	setNotificationStatusWithExpiry = function( key, ttl ) {
+		var now = new Date();
+		localStorage.setItem( key, now.getTime() + ttl );
+	},
+
+	getNotificationStatusWithExpiry = function( key ) {
+		var expiryTimeValue = localStorage.getItem( key );
+		var now = new Date();
+
+		// compare the expiry time of the item with the current time
+		if ( now.getTime() > expiryTimeValue ) {
+
+			// If the item is expired, delete the item from storage
+			// and return null
+			localStorage.removeItem( key );
+			return null;
+		}
+		return expiryTimeValue;
+	},
+
+
+/**
+	 * @method init
+	 * @param {jQuery Event} event Event that triggered the function call
+	 */
+	init = function( event ) {
+
+		// Start initialization
+		// returns DOM object = proceed with init
+		// returns undefined = do not proceed with init (e.g., already initialized)
+		var elm = wb.init( event, componentName, selector ),
+			$elm;
+		if ( elm ) {
+			$elm = $( elm );
+
+			// Initiate dictionary
+			i18nDict = i18nDict[ $( "html" ).attr( "lang" ) || "en" ];
+			i18nDict = {
+				skip: i18nDict[ "bubble-skip" ],
+				closeNotif: i18nDict[ "bubble-closeNotif" ]
+			};
```

Comment:
No need to shorten the dictionary names, I would recommend you to simply remove "bubble-" from the get-go (at line 23-24 and 27-28) so you can stick to **i18nDict.skip** and **i18nDict.closeNotif**.